spring/src/main/java/io/seata/spring/annotation/GlobalTransactional.java

```diff
@@ -33,10 +32,11 @@
 
     /**
      * Global transaction timeoutMills in MILLISECONDS.
+     * The default value is the config: client.tm.global-transaction-timeout
      *
      * @return timeoutMills in MILLISECONDS.
      */
-    int timeoutMills() default TransactionInfo.DEFAULT_TIME_OUT;
+    int timeoutMills() default -1;
```

Comment:
not elegant.

Reply:
default 0?

Reply:
or a constant, with name: DEFAULT_VALUE_FROM_CONFIG？